docs/recipes/recipes.mdx

```diff
@@ -183,15 +183,16 @@ const store = createStore(() => ({ ... }))
 const { getState, setState, subscribe, destroy } = store
 ```
 
-You can even consume an existing vanilla store with React:
+You can use a vanilla store with `useStore` hook available since v4.
```

Comment:
I think the docs should implicitly be for the current version of Zustand.
Also, I'll mention that we are using a vanilla store _in React_, because this section is all about the usage _without React_.

```suggestion
You can use a vanilla store in React with a `useStore` hook.
```